src/utils/revurdering/revurderingUtils.ts

```diff
@@ -128,3 +137,20 @@ export const finnNesteRevurderingsteg = (
 
     return førsteIkkeVurderteSteg ?? RevurderingSteg.Oppsummering;
 };
+
+export const getAvsluttedeOgIkkeAvsluttedeRevurderinger = (
+    revurderinger: AbstraktRevurdering[]
+): { avsluttedeRevurderinger: AbstraktRevurdering[]; åpneRevurderinger: AbstraktRevurdering[] } => {
+    const avsluttedeRevurderinger: AbstraktRevurdering[] = [];
+    const åpneRevurderinger: AbstraktRevurdering[] = [];
+
+    revurderinger.forEach((r) => {
```

Comment:
Ville unngått `forEach` og `push` for å hindre mutability. Kunne man enten kjørt to filters, (eller èn reduce om man ikke vil loope to ganger. Frykter dog at det ikke blir så lesbart)?
```
const avsluttedeRevurderinger: AbstraktRevurdering[] = revurderinger.filter((r) => erAbstraktRevurderingAvsluttet(r));
const åpneRevurderinger: AbstraktRevurdering[] = revurderinger.filter((r) => !erAbstraktRevurderingAvsluttet(r));
```

Reply:
endret hele greien :)